src/pages/who-we-are/_OurOffices.tsx

```diff
@@ -126,7 +126,7 @@ const OurOffices = () => {
 
             <NumberSection columns="1fr 1fr 1fr 1fr" column_gap="120px" row_gap="4rem">
                 {our_offices_count.map(({ count, title }) => (
-                    <StyledFlex fd="column" key={title}>
+                    <StyledFlex fd="column" key={title.props.translate_text}>
```

Comment:
ditto

Reply:
@hirad-rewok these data are static, I think we don't need optional chaining here